psi4/src/psi4/dfocc/mp3_W_intr.cc

```diff
@@ -31,6 +31,8 @@
 #include "dfocc.h"
 
 using namespace psi;
+using namespace std;
```

Comment:
We've purged `using namespace std` from the rest of the code for clarity. So please go ahead and `std::` anything that needs it.